infrastructure/material_list/values.yaml

```diff
@@ -0,0 +1,49 @@
+# Default values for cover-service.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+env: "prod"
+
+ingress:
+  enabled: true
+  enableAuth: false
+  enableTLS: true
+  cert: "letsencrypt"
+  path: "/"
+
+hpa:
+  enabled: true
+  minReplicas: 2
+  maxReplicas: 5
+  cpuTargetAverageUtilization: 70
+  memoryTargetAverageUtilization: 80
+
+app:
+  image:
+    version: "latest"
+    imagePullPolicy: "IfNotPresent"
+  db:
+    host: 'material-list-database.mariadb.database.azure.com'
+    port: '3306'
+    database: 'material_list'
+    ssl: 'true'
+  adgangsplatformen:
+    driver: 'production'
+  log_channel: 'stderr'
+  url: 'prod.materiallist.dandigbib.org'
+  timezone: 'UTC'
+  cacheDriver: 'file'
+  queueConnection: 'sync'
+  cors_allow_origin: "*"
+  php:
+    max_children: "80"
+    start_servers: "15"
+    min_spare_servers: "15"
+    max_spare_servers: "30"
+    slowlog_timeout: "1"
```

Comment:
These values are a lot higher than the current values for production from the deleted `zz-pm.conf`. I do not know whether that is intentional or not.

Reply:
The values was based on requests to cover service... adjusted them to match the old values, but will be re-adjusted when we get load on the setup.